app/controllers/posts_controller.rb

```diff
@@ -52,7 +75,10 @@ def update
   end
 
   def destroy
-    @post.destroy
+    @council = Council.find_by_url!(params[:council_id])
+    @post = @council.posts.find(params[:id])
```

Comment:
I was thinking that I do not want to allow destroying a `@post` if it is not owned by the given `@council`.
Otherwise I agree.